### PR TITLE
Push code-server image to GHCR

### LINKED ISSUE #3335: Provide alternate image registry to dockerhub
Hello,

In order to combat dockerhub's imposed rate limits it would be great to host these images on ghcr or quay.io. Are there on any thoughts from the community on this initiation?

### PR DESCRIPTION
**Pull request by onedr0p**
Fixes #3335

### CODE REVIEW

**Review by code-asher — APPROVED** [review]
Awesome!!  Thanks for taking care of this!

File: .github/workflows/docker.yaml [hunk 1]
```diff
@@ -35,6 +35,12 @@ jobs:
           username: ${{ secrets.DOCKER_USERNAME }}
           password: ${{ secrets.DOCKER_PASSWORD }}
 
+      - name: Login to GHCR
+        uses: docker/login-action@v2
+        with:
+          username: ${{ github.actor }}
```
Comment (code-asher) [inline .github/workflows/docker.yaml:41]: Whoops, I missed this.  Do we need `registry: ghcr.io` as well?  Not sure if it tries to log into docker.io by default or not

**Comment by onedr0p** [comment]
I think there might need to be some action on the maintainer side for the coder org, you may need to enable packages, see:

image

File: .github/workflows/docker.yaml (hunk 1 above)
Comment (onedr0p) [reply .github/workflows/docker.yaml:42]: Yep, added!

**Pull request merged**